DigitalLearningSolutions.Web.Tests/Controllers/Login/LoginControllerTests.cs

```diff
@@ -144,5 +144,44 @@ public void Unapproved_delegate_account_redirect_to_not_approved_page()
             // Then
             result.Should().BeViewResult().WithViewName("AccountNotApproved");
         }
+
+        [Test]
+        public void Log_in_with_admin_id_recovers_associated_delegate_users()
```

Comment:
I think "recovers" is a little bit odd wording for this, maybe "fetches" associated delegate users?